code/modules/jobs/job_types/trusted_players/sephirah.dm

```diff
@@ -9,12 +9,14 @@
 	access = list(ACCESS_NETWORK, ACCESS_COMMAND, ACCESS_MANAGER) // Network is the trusted chat gamer access
 	minimal_access = list(ACCESS_NETWORK, ACCESS_COMMAND, ACCESS_MANAGER)
 	mapexclude = list("wonderlabs", "mini")
-	job_important = "You are a roleplay role, and may not partake in combat. Assist the manager and roleplay with the agents and clerks"
-	job_notice = "In the OOC tab you have a verb called 'randomize current abnormality'. \
-		It is to be used to spice up boring rounds, and punish manager players you think are playing too safe. \
-		This is an OOC tool. Do not bring alert to the fact that you can do this IC. Alert any administrators if any IC action is taken against you. \
+	job_important = "You are a strictly roleplay role and are forbidden from partaking in combat. Assist the Manager and improve IC roleplay during the round."
+	job_notice = "In the OOC tab you have a verb called 'Randomize Current Abnormality'. \
+		It should be used to spice up boring rounds and discourage Managers you think are playing too safe. \
+		This is an OOC-only tool. Do not allow anyone IC to learn of this ability. Alert administrators if any IC action is taken against you. \
 		Abusing this will result in a loss of whitelist."
 
+	job_abbreviation = "SEPH"
```

Comment:
This one isn't needed; Sephirah have a unique naming scheme and a robotic voice.